.github/workflows/doc.yml

```diff
@@ -0,0 +1,17 @@
+# Qserv CI workflow
+# FIXME: Github action runner are too weak to run e2e tests
```

Comment:
👍 ...we should discuss (maybe at the next cowork?) where integration tests should be run; one of the benefits of lite is a much faster build time, and so the integration tests can be run on GitHub actions during the lite build. It might make sense to run them on the operator as well, to test for correctness.